packages/web/components/layouts/main.tsx

```diff
@@ -60,7 +56,6 @@ export const MainLayout = observer(
         </div>
         <NavBar
           className="ml-sidebar md:ml-0"
-          title={selectedMenuItem?.label ?? ""}
```

Comment:
i think we don't need to remove every title on every page, you can use the `useNavbar` hook inside the selected page to manipulate the current title